public/assets/stylesheets/gameBackground.css

```diff
@@ -0,0 +1,38 @@
+.game-background {
+  background: linear-gradient(#B9B9B9,#E5E5E5)
+}
+
+.title-image {
+  display: block;
+  margin: auto;
+  padding-top: 35px;
+}
+
+.button-background {
+  background-color: Transparent;
+  border-style: solid;
+  border-color: darkgray;
+  border-radius: 100px;
+}
+
+.left-column {
+  margin-top: 3%;
+  margin-left: 10%;
+  float: left;
+}
+
+.right-column {
+  float: right;
+  margin-top: 3%;
+  margin-right: 10%;
+}
+
+.test {
+  background-color: black;
+  float: left;
+  margin-left: 35%;
+}
+
+.title-spacing {
+  margin-bottom: 60px;
+}
```

Comment:
Something weird is going on with the spacing.  The board is not centered for all screen sizes.  That should be fixed.

It doesn't need to be centered, but it does need to have some sort of visually pleasing location.